File: Bars.GkhDI/Entities/Service/WorkCapRepair.cs
namespace Bars.GkhDi.Entities
{
    using Bars.Gkh.Entities;
    using Bars.Gkh.Entities.Dicts;

    /// <summary>
    /// Работа услуги капремонт
    /// </summary>
    public class WorkCapRepair : BaseGkhEntity
    {
        /// <summary>
        /// Услуга
        /// </summary>
        public virtual BaseService BaseService { get; set; }

        /// <summary>
        /// Работа капремонта
        /// </summary>
        public virtual Work Work { get; set; }

        /// <summary>
        /// Запланированный объем
        /// </summary>
        public virtual decimal? PlannedVolume { get; set; }

        /// <summary>
        /// Запланированная сумма
        /// </summary>
        public virtual decimal? PlannedCost { get; set; }

        /// <summary>
        /// Фактический объем
        /// </summary>
        public virtual decimal? FactedVolume { get; set; }

        /// <summary>
        /// Фактическая сумма
        /// </summary>
        public virtual decimal? FactedCost { get; set; }
    }
}

File: Bars.GkhDI/libs/B4/model/Service/WorkCapRepair.js
Ext.define('B4.model.service.WorkCapRepair', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'WorkCapRepair'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'BaseService', defaultValue: null },
        { name: 'Work', defaultValue: null },
        { name: 'PlannedVolume', defaultValue: null },
        { name: 'PlannedCost', defaultValue: null },
        { name: 'FactedVolume', defaultValue: null },
        { name: 'FactedCost', defaultValue: null },
        { name: 'Name' },
        { name: 'WorkId' }
    ]
});